README.md

```diff
@@ -175,8 +175,10 @@ Parses a given xlsx file inside the `server/scripts/xlsx/` folder and feeds eith
 
 If you've spun up the application using Docker Compose i.e. `make local-run`, you can run the site seeding script with either of the following make commands: 
 
-- `make feed-participants my_spreadsheet.xlsx`
-- `make feed-sites my_spreadsheet.xlsx`
+- `make seed-participants my_spreadsheet.xlsx`
+- `make seed-sites my_spreadsheet.xlsx`
+
+N.B. - The sample seed files are available [here](https://drive.google.com/drive/folders/17sXAQObfjK3KSF70j6etoI--1UYXyqlx?usp=sharing)
```

Comment:
Is this the right place for private links? I'm thinking possibly pinned post in Slack or the Project Wiki

Reply:
Project wiki or slack is more appropriate place but people has tendency to use Readme to setup env.

Reply:
can this not be checked in and versioned, so we dont have to download, keep tabs in slack.  Seems like its just few kb of data

Reply:
If those files contain any PI or Sensitive info then `NO`.

Reply:
But we can generate a good randomise seed method or data for this.